src/error.rs

```diff
@@ -11,13 +11,15 @@ use cosmian_crypto_base::CryptoBaseError;
 use hex::FromHexError;
 use thiserror::Error;
 
-#[derive(Error, Debug, Clone, PartialEq)]
+#[derive(Error, Debug)]
 pub enum FormatErr {
     //
     // External errors conversion
     #[error("{0}")]
     CryptoError(String),
     #[error(transparent)]
+    PolicyError(#[from] abe_policy::error::Error),
```

Comment:
I guess now we can remove some of the variants we have below, which are redundant with `abe_policy`'s error.

Also, let's expose `Error` at top level of `abe_policy`'s crate, so you can `use abe_policy::Error` without having to know how much deeper is the `Error` struct within `abe_policy`'s crate (if you move it internally, it's gonna be a breaking change for nothing basically)

Reply:
Done